src/frontend/src/pages/TeamsPage/TeamSummary.tsx

```diff
@@ -35,8 +38,10 @@ const TeamSummary: React.FC<TeamSummaryProps> = ({ team }) => {
         </Typography>
       </CardContent>
       <CardActions>
-        <Button variant="outlined" size="small" component={Link} href={`${routes.TEAMS}/${team.teamId}`}>
-          See More
+        <Button variant="outlined" size="small">
```

Comment:
Can you change this to contained?

Reply:
I'd say just leave it alone for now tbh; when you switch it to contained, it makes the coloring messed up (it's red on red), might be good to bring up in another easy ticket for a new dev if we wanna change the look of this button, I know there's other stuff on this page that needs to be cleaned up looks-wise like that that's def out of scope for this ticket